Tools/JFusion/Program.cs

```diff
@@ -30,44 +30,101 @@ public static void Main(string[] args)
+                case ConsoleKey.D1:
+                    MFAPATH:
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    string mfapath = string.Empty;
+                    Console.Write("MFA path: ");
+                    mfapath = Console.ReadLine().Trim('"');
+                    if (!File.Exists(mfapath))
+                    {
+                        Console.ForegroundColor = ConsoleColor.Red;
+                        Console.WriteLine("ERROR: File not found");
+                        goto MFAPATH;
+                    }
+                    string JOutN = string.Empty;
+                    Console.WriteLine("Output Name: ");
+                    JOutN = Console.ReadLine();
+                    string JOutP = string.Empty;
+                    Console.WriteLine("Output Path: ");
+                    JOutP = Console.ReadLine().TrimEnd('\\');
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    Console.WriteLine("Converting MFA to Json");
+                    Console.WriteLine("Reading MFA...");
                     var newMfa = new MFAData();
-                    newMfa.Read(new ByteReader("snakeinmyass.mfa", FileMode.Open));
-                    Console.WriteLine("MFA reading finished. Converting MFA...");
+                    newMfa.Read(new ByteReader(mfapath, FileMode.Open));
+                    Console.WriteLine("MFA reading finished.");
+                    Console.WriteLine("Converting MFA...");
                     var mfaFProj = JMFAFile.FromMFA(newMfa);
-                    Console.WriteLine("MFA conversion finished. Writing Json...");
-                    mfaFProj.Write("Tests\\TestFProject");
+                    Console.WriteLine("MFA conversion finished.");
+                    Console.WriteLine("Writing Json...");
+                    mfaFProj.Write($"{JOutP}\\{JOutN}");
                     Console.WriteLine("Json writing finished");
-                    break;
-                case ConsoleKey.D2:
-                    var ogFProj = JMFAFile.Open("Tests\\TestFProject\\Five Nights at Freddy's.json");
-                    var mfa = ogFProj.ToMFA();
-                    mfa.Write(new ByteWriter("my_ass.mfa", FileMode.Create));
-                    break;
+                    break;
+                
+
+                case ConsoleKey.D2:
+                    JSONPATH:
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    string jpath = string.Empty;
+                    Console.Write("Json path: ");
+                    jpath = Console.ReadLine().Trim('"');
+                    if (!File.Exists(jpath))
+                    {
+                        Console.ForegroundColor = ConsoleColor.Red;
+                        Console.WriteLine("ERROR: Project not found");
+                        goto JSONPATH;
+                    }
+                    MFAOUT:

```

Comment:
Can we avoid using goto statements? Wrap into while loop or into separate function

Reply:
idk i saw kostya using them so i kept using them

Reply:
> Can we avoid using goto statements? Wrap into while loop or into separate function

Done.